### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by philipp94831**

### CODE REVIEW

File: common/src/main/java/com/bakdata/dedupe/similarity/CommonSimilarityMeasures.java [hunk 1]
```diff
@@ -121,8 +121,8 @@ public boolean isSymmetric() {
             }
 
             @Override
-            public double getNonNullSimilarity(@NonNull T left, @NonNull T right, @NonNull SimilarityContext context) {
-                return new JaroWinklerDistance().apply(left, right).doubleValue();
+            public double getNonNullSimilarity(@NonNull final T left, @NonNull final T right, @NonNull final SimilarityContext context) {
+                return new JaroWinklerSimilarity().apply(left, right).doubleValue();
```
Comment (philipp94831) [inline common/src/main/java/com/bakdata/dedupe/similarity/CommonSimilarityMeasures.java:125]: https://issues.apache.org/jira/browse/TEXT-191

File: azure-pipelines.yml [hunk 2]
```diff
@@ -12,16 +12,20 @@ variables:
...
   - group: sign
   - group: ossrh
+  - group: changelog
 
 resources:
   repositories:
     - repository: templates
       type: github
       name: bakdata/bakdata-project-templates
+      ref: tmp/upload-snapshot
```
Comment (JakobEdding) [inline azure-pipelines.yml:22]: TODO

**Review by JakobEdding — CHANGES_REQUESTED** [review]

File: azure-pipelines.yml (hunk 2 above)
Comment (philipp94831) [reply azure-pipelines.yml]: I always remember to late 😬 Updated already! :D

Comment (JakobEdding) [reply azure-pipelines.yml]: That's what reviews are for :D

**Review by JakobEdding — APPROVED** [review]

**Pull request merged**